CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## v0.6.1 / 2017-04-28
+
+- [BUGFIX] Fix broken OpsGenie integration
```

Comment:
The fix was more general for all integrations using a hash for deduplication. Just that OpsGenie is the only reported one so far.